### PR TITLE
Specified two-column csv file

### LINKED ISSUE #76: Better documentation and error handling on two-column CSV file limitation
It's hardly visible from the documentation that funnel only operates on two-column CSV files. Furthermore, nothing is logged to c_funnel.log if multi-column CSV files are passed to funnel. In my case I wondered about the unexpected output CSV files with only one row.

1. It should be maded transparent (in README and help text) that only two-column CSV files are handled.
2. The error reporting should be improved for multi-column CSV files.
3. Are there any plans to support multi-column CSV files?

Thank you!

### PR DESCRIPTION
**Pull request by JayHuLBL**
This closes #76.

### CODE REVIEW

File: pyfunnel/pyfunnel.py [hunk 1]
```diff
@@ -89,6 +89,8 @@
         data[s] = dict(x=[], y=[])
         with open(vars(args)[s]) as csvfile:
             spamreader = csv.reader(csvfile)
+            if (len(next(spamreader)) != 2):
```
Comment (AntoineGautier) [inline pyfunnel/pyfunnel.py:92]: I believe that if you use `next` here, the next iteration on the iterator (for loop below) starts at index 1 and not 0, i.e., the first row is skipped.
Why not test `len(spamreader[0])`? Or each line within the for loop: `len(row)`?

**Review by mwetter — APPROVED** [review]

File: pyfunnel/pyfunnel.py [hunk 2]
```diff
@@ -89,6 +89,8 @@
...
+                raise RuntimeError("The {} CSV file must have exactly two columns.".format(s))
```
Comment (AntoineGautier) [inline pyfunnel/pyfunnel.py:93]: Shouldn't this rather be an `IOError `?

**Review by AntoineGautier — CHANGES_REQUESTED** [review]
Please see my inline comments.

**Comment by JayHuLBL** [comment]
@AntoineGautier Please see the updated one.

**Comment by AntoineGautier** [comment]
@JayHuLBL In 857ddda I propose to keep the iterable object (rather than casting it into a list which is potentially less memory-efficient) and check the length of each row. My thinking is that, if the CSV is corrupted with, e.g., an element missing at row 10000, the user will have better error information this way. If you want to revert back to checking only the first row, maybe keeping the iterable object and checking `len(list(spamreader)[0])` would be best.

**Review by AntoineGautier — APPROVED** [review]

**Pull request merged**